.github/workflows/alpha.yml

```diff
@@ -77,6 +77,14 @@ jobs:
           COSIGN_PWD: ${{ secrets.COSIGN_PWD }}
           GORELEASER_CURRENT_TAG: v0.0.0-alpha
 
+      - name: Delete outdate release
```

Comment:
I notice that the [alpha](https://github.com/ZupIT/horusec/tree/alpha) tag is not being synchronized with main, why do wee need delete the release? Synchronize the alpha tag with main and update the release it is not enough?

Reply:
It would be more work and we would have the same result.